alma/controllers/hook/FrontHeaderHookController.php

```diff
@@ -316,6 +316,8 @@ public function manageInsuranceAssetsAfter17()
                 ConstantsHelper::INSURANCE_PRODUCT_CSS_ID,
                 "modules/$this->moduleName/" . ConstantsHelper::INSURANCE_PRODUCT_CSS_PATH
             );
+
+            return '<script type="module" src="https://protect.sandbox.almapay.com/displayModal.js"></script><div id="alma-insurance-modal"></div>';
```

Comment:
une constante serait mieux pr stocket l'url

Reply:
@ClaireAlma 
Yes, It was for the explo. It's why the PR is still in Draft :)